tools/build_chain.sh

```diff
@@ -744,6 +744,7 @@ if [ ! -z \${node_pid} ];then
     exit 0
 else 
     ${start_cmd} &
+    echo "start time: "\$(date '+%Y-%m-%d %H:%M:%S') >> nohup.out
```

Comment:
once fisco-bcos start, it will output start log. Please do this in that.
Please delete this from start.sh